pyproject.toml

```
@@ -28,10 +28,10 @@ dependencies = [
     "aleph-message==0.4.9",
     "aleph-p2p-client@git+https://github.com/aleph-im/p2p-service-client-python@2c04af39c566217f629fd89505ffc3270fba8676",
     "aleph-nuls2==0.1.0",
-    "aleph-pytezos@git+https://github.com/aleph-im/aleph-pytezos.git@32dd1749a4773da494275709060632cbeba9a51b",
+    "aleph-pytezos@git+https://github.com/aleph-im/aleph-pytezos.git@hoh-continue-upgrade",
```

Can we use it from PyPI ?

```suggestion
    "aleph-pytezos==3.13.4",
```

---

Yup, it's finally released